tests/spec/core/markdown-spec.js

```diff
@@ -41,69 +45,86 @@ describe("Core - Markdown", function() {
     };
     ops.config.format = "markdown";
     makeRSDoc(ops, function(doc) {
-      expect($(".note em", doc).length).toEqual(1);
-      expect($(".issue em", doc).length).toEqual(1);
-      expect($(".req em", doc).length).toEqual(1);
-      expect($(".req h3", doc).length).toEqual(1);
+      expect(doc.querySelector(".note em")).toBeTruthy();
+      expect(doc.querySelector(".issue em")).toBeTruthy();
+      expect(doc.querySelector(".req em")).toBeTruthy();
+      expect(doc.querySelector(".req h3")).toBeTruthy();
     }).then(done);
   });
 
   it("should remove left padding before processing markdown content", function(done) {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\n    Foo\n    ===\n",
+       "\n\n  Foo\n  ===\n    * list item 1\n    * list item 2\n      * nested list item",
```

Comment:
What bothers me is that this test was making sure that 4 spaced indented content was treated as a massive code block.

Given this is now a 2 space indent, you're no longer testing for the same thing.